src/gui/properties/speedplotview.cpp

```diff
@@ -169,6 +170,67 @@ int SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
+{
+    // table of supposed nice steps for grid marks to get nice looking quarters of scale
+    const double roundingTable[] = {1.2, 1.6, 2, 2.4, 2.8, 3.2, 4, 6, 8, 10};
+    qint64 getNiceYScale(double value)
+    {
+        if ( value == 0.0  ) return 0;
+        if ( value <= 12.0 ) return 12;
+        qint64 scale = 1;
+        while (value > 1024) {
+            value /= 1024;
+            scale *= 1024;
+        }
+        if (value > 100.0) {
+            int roundedValue = (static_cast <int> (value / 40)) * 40;
+            while (roundedValue < value)
+                roundedValue += 40;
+            return scale * roundedValue;
+        }
+        if (value > 10.0) {
+            int roundedValue = (static_cast <int> (value / 4)) * 4;
+            while (roundedValue < value)
+                roundedValue += 4;
+            return scale * roundedValue;
+        }
+        while (value > 10) {
+            value /= 10;
+            scale *= 10;
+        }
+        const size_t ROUNDING_TABLE_SIZE = sizeof(roundingTable) / sizeof(roundingTable[0]);
+        for (size_t i = 0; i < ROUNDING_TABLE_SIZE; ++i) {
+            if (value <= roundingTable[i]) {
+                return qint64(roundingTable[i] * scale);
+            }
+        }
+        return scale;
+    }
+
+    using Utils::Misc::SizeUnit;
+
+    QString niceLabel(qint64 bytesValue, SizeUnit &unit, bool fixed)
+    {
+        qreal friendlyVal;
+        if (fixed) {
+            // calculate friendlyVal on fixed unit size from bytesVal
+            friendlyVal = qreal(bytesValue) / Utils::Misc::sizeInBytes(1, unit);
+        } else {
```

Comment:
`else` should be in the next line.
You can write it as `else if (...)` in this case.